### PR TITLE
Fixes a few cases when you can get stuck in your infernal jaunt.

### LINKED ISSUE #17742: Devils who go into the archdemon phaze while jaunting cannot leave the jaunt
archdemon is broken tbh

### PR DESCRIPTION
**Pull request by lordpidey**
Fixes #17742

### CODE REVIEW

File: code/game/gamemodes/demon/demoninfo.dm [hunk 1]
```diff
@@ -182,6 +182,8 @@ var/global/list/lawlorify = list (
 		H.set_species(/datum/species/human, 1)
 		H.regenerate_icons()
 	give_base_spells()
+	if(istype(owner.current.loc, /obj/effect/dummy/slaughter/))
+			owner.current.forceMove(get_turf(owner.current))//Fixes dying while jaunted leaving you permajaunted.
```
Comment (KazeEspada) [inline code/game/gamemodes/demon/demoninfo.dm]: COLON DETECTED!

Comment (RemieRichards) [inline code/game/gamemodes/demon/demoninfo.dm]: uh, there's no colon here.

Comment (Man-Shroom-) [inline code/game/gamemodes/demon/demoninfo.dm]: kaze are you feeling well

Comment (RemieRichards) [inline code/game/gamemodes/demon/demoninfo.dm]: I think kaze is broken.

**Comment by KazeEspada** [comment]
I blame the pain meds.

**Comment by coiax** [comment]
Does this include falling unconscious from morphine overdose

**Comment by lordpidey** [comment]
If I'm not mistaken, you should simply wake back up eventually, and be able to phase in.

**Comment by optimumtact** [comment]
This is why jaunt is crap btw, so snowflake, the sooner we get a systme the better :cry:

**Comment by phil235** [comment]
those checks are a bit ugly tbh.

**Comment by optimumtact** [comment]
jaunt is ugly as well, it's basically just stuffing the user into a holder obj.

It's gross

**Pull request merged**